packages/business-rules/src/schemas/householder-appeal/generic-validators/boolean-validation.js

```diff
@@ -0,0 +1,7 @@
+const pinsYup = require('../../../lib/pins-yup');
```

Comment:
can you move generic validators into - say - the src/schemas/components folder so they can be used for full appeal  schema too?